pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -738,20 +735,18 @@ def animate_fields_from_aedtplt(
 
         Parameters
         ----------
-        plotname : str
+        plot_name : str
             Name of the plot or the name of the object.
         plot_folder : str, optional
             Name of the folder in which the plot resides. The default
             is ``None``.
         variation_variable : str, optional
             Variable to vary. The default is ``"Phase"``.
-        variation_list : list, optional
+        variations : list, optional
             List of variation values with units. The default is
             ``["0deg"]``.
         project_path : str, optional
             Path for the export. The default is ``""`` which export file in working_directory.
-        meshplot : bool, optional
-             The default is ``False``. Valid from Version 2021.2.
         export_gif : bool, optional
              The default is ``False``.
                 show=False,
```

Comment:
```suggestion
             Whether to export the GIF file. The default is ``False``.
                show=False,
```
I'm not sure why "show=False," is there. I believe this should be deleted.